src/utils/wallet.js

```diff
@@ -577,16 +582,22 @@ class Wallet {
     }
 
     async initializeRecoveryMethod(accountId, method, isNew) {
+        const { seedPhrase } = generateSeedPhrase()
+
         if (isNew) {
-            return await sendJson('POST', ACCOUNT_HELPER_URL + '/account/initializeRecoveryMethodForTempAccount', {
+            await sendJson('POST', ACCOUNT_HELPER_URL + '/account/initializeRecoveryMethodForTempAccount', {
                 accountId,
                 method,
+                seedPhrase: [seedPhrase]
```

Comment:
Can you not pass in array? Just use:
```
...
seedPhrase,
...
```

Reply:
![Screen Shot 2020-08-24 at 10 44 18 PM](https://user-images.githubusercontent.com/24921205/91127562-5aeb3d80-e65b-11ea-96a4-8a465326623d.png)

@mattlockyer did I misunderstand you?